presto-product-tests/src/main/resources/com/facebook/presto/tests/jdbc/get_nation_columns_simba.result

```diff
@@ -1,5 +1,5 @@
 -- delimiter: |; ignoreOrder: false; ignoreExcessRows: false;
 hive|default|nation|n_nationkey|-5|bigint|19|8|0|10|1|null|null|null|null|null|1|YES|null|null|null|null|null|null|
-hive|default|nation|n_name|12|varchar|2048|2048|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|null|
+hive|default|nation|n_name|12|varchar(25)|25|25|null|10|1|null|null|null|null|2048|2|YES|null|null|null|null|null|null|
```

Comment:
is varchar(25) what is supposed to be printed out for this metadata? or is it varchar with a separate field being 25 (precision)?